spring/README.md

```diff
@@ -32,13 +50,34 @@ import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ComponentScan("com.example.app")
+@ComponentScan("cucumber.runtime.java.spring.beans")
```

Comment:
This should not be changed. The `com.example.app` should be used for consistency everywhere.